pyproject.toml

```diff
@@ -1,3 +1,70 @@
 [build-system]
-requires = ["setuptools"]
+requires = ["setuptools >= 61.0"]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "xmlschema"
+version = "3.4.3"
+description = "An XML Schema validator and decoder"
+readme = "README.rst"
+license = {text = "MIT"}
```

Comment:
`license` and `license-files` is about to change with PEP 639.

Reply:
ok, but still a referral to `LICENSE` file is needed, to link copyright information

Reply:
Will add it, I could not build the package yesterday, due to confusion about license_file, license_files and license-files in documentation on packaging with setuptools.

Reply:
I added it explicitly, but if I understand it correctly, LICENSE is added by default.

Ref: https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#setuptools-specific-configuration